Dimer_scf_vs_radius.py

```diff
@@ -0,0 +1,42 @@
+from wrap_pyscf import *
+
+import pickle
+
+scf_guess = {}
+
+filename = "scf_record"
```

Comment:
I recommend having the filename actually be set to what you are going to store the file as. This avoids needing to remember to modify the script/rename the file on subsequent runs.